CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+# dbt_klaviyo v0.7.2
+[PR #38](https://github.com/fivetran/dbt_klaviyo/pull/38) includes the following updates:
+
+## Bug Fixes
+- Removes `not_null` tests for `person_id` from select models. This is because Klaviyo can record events with deactivated profiles, resulting in null `person_id`s. Therefore models built off of tables with event-based grains may have null `person_id`s.
```

Comment:
Can we list all the models this test was removed from. This way it is clear to users upgrading which models this is impacting directly.

Reply:
done! Also did docs regen and made sure the tests have been removed. And added the auto releaser. once buildkite passes I'll kick off the release process